### PR TITLE
PROXY-9: Add parameters to the query 

### PR DESCRIPTION
**Pull request by DmitryPopov1988**

### CODE REVIEW

File: src/service/nasa_meteor_service.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        .get(config.nasaApiConfig.url, {
+            params: {
+                start_date: requestStartDate,
+                end_date: requestEndDate,
+                api_key: config.nasaApiConfig.key,
+            },
+        })
+        .then((response) => {
+                if (countOnly) {
+                    return {count: countMeteors(response.data)};
```
Comment (mariaYurcheuskaya) [inline src/service/nasa_meteor_service.js]: what do you think about moving response.data to the constant as it is used 3 times

**Review by mariaYurcheuskaya — APPROVED** [review]

File: src/service/nasa_meteor_service.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                }
+                if (wereDangerous) {
```
Comment (plutsenka) [inline src/service/nasa_meteor_service.js]: What if both - countOnly & wereDangerous params are presented in request?

File: src/service/nasa_meteor_service.js (hunk 1 above)
Comment (DmitryPopov1988) [reply src/service/nasa_meteor_service.js]: Good spot. Thanks, mate.
Updated.

File: src/service/nasa_meteor_service.js (hunk 2 above)
Comment (DmitryPopov1988) [reply src/service/nasa_meteor_service.js]: Thank you for noticing. Fixed.

**Pull request merged**